onsstove/technology.py

```diff
@@ -150,59 +148,38 @@ def mortality(self, specs_file, paf_0_alri, paf_0_copd, paf_0_lc, paf_0_ihd):
         paf_ihd = self.paf(rr_ihd, 1 - specs_file['clean_cooking_access'])
         paf_lc = self.paf(rr_lc, 1 - specs_file['clean_cooking_access'])
 
-        mort_alri_U = specs_file["Urban_HHsize"] * (paf_0_alri - paf_alri) * specs_file["Mort_ALRI"]
-        mort_copd_U = specs_file["Urban_HHsize"] * (paf_0_copd - paf_copd) * specs_file["Mort_COPD"]
-        mort_ihd_U = specs_file["Urban_HHsize"] * (paf_0_ihd - paf_ihd) * specs_file["Mort_IHD"]
-        mort_lc_U = specs_file["Urban_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
-
-        mort_alri_R = specs_file["Rural_HHsize"] * (paf_0_alri - paf_alri) * specs_file["Mort_ALRI"]
-        mort_copd_R = specs_file["Rural_HHsize"] * (paf_0_copd - paf_copd) * specs_file["Mort_COPD"]
-        mort_ihd_R = specs_file["Rural_HHsize"] * (paf_0_ihd - paf_ihd) * specs_file["Mort_IHD"]
-        mort_lc_R = specs_file["Rural_HHsize"] * (paf_0_lc - paf_lc) * specs_file["Mort_LC"]
+        mort_alri = specs_file["Population_start_year"] * (paf_0_alri - paf_alri) * (specs_file["Mort_ALRI"]/100000)
+        mort_copd = specs_file["Population_start_year"] * (paf_0_copd - paf_copd) * (specs_file["Mort_COPD"]/100000)
+        mort_ihd = specs_file["Population_start_year"] * (paf_0_ihd - paf_ihd) * (specs_file["Mort_IHD"]/100000)
+        mort_lc = specs_file["Population_start_year"] * (paf_0_lc - paf_lc) * (specs_file["Mort_LC"]/100000)
```

Comment:
If we pass the main `gdf` as an input and change `specs_file["Population_start_year"]` for `gdf['Calibrated_pop']` we would get the mort and morb for each cell right?

Reply:
yeah that is true, you want us to change to that instead? both here and in morb?

Reply:
And then I assume we need to sum it for the equations below that

Reply:
Yes I think so